doc/api/fs.md

```diff
@@ -5791,14 +5815,28 @@ changes:
 -->
 
 * `fd` {integer}
-* `buffer` {Buffer|TypedArray|DataView|string|Object}
+* `buffer` {Buffer|TypedArray|DataView}
```

Comment:
Here as well https://github.com/nodejs/node/blob/ae1f6db0886a540dcf33f27bb4f13c37e0c8899e/doc/api/fs.md#L5806-L5814

Reply:
Resolved in the same way as [filehandle.write](https://github.com/nodejs/node/pull/41677#discussion_r841283211): redundant changelog entry is removed, string-related changes were already added for `fs.writeSync(fd, string, ...)` version.